.env.elves

```
@@ -0,0 +1,10 @@
+VITE_DOC_ID=elves
+VITE_DOC_NAME=ELVES Paper
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)